accounting/accounting_cli_functions.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import sqlite3
+import argparse
+import time
+import sys
+
+import pandas as pd
+
+def view_user(user):
+    try:
+        # open connection to database
+        conn = sqlite3.connect("FluxAccounting.db")
+        # get the information pertaining to a user in the Accounting DB
+        select_stmt = (
+            "SELECT * FROM association_table where user_name='" + user + "'"
+        )
+        dataframe = pd.read_sql_query(select_stmt, conn)
```

Comment:
Looks like pandas also supports secure parameter substitution in a similar way [[1]](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_sql_query.html):

```suggestion
        select_stmt = (
            "SELECT * FROM association_table where user_name='?'"
        )
        dataframe = pd.read_sql_query(select_stmt, conn, params=(user,))
```